submissions/AndriyM/hooli-style-popup/index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="./styles.css" />
+        <title>Html-css-popup</title>
+    </head>
+    <body>
+        <div class="wrapper">
+            <header class="header">
+                <div class="container">
+                    <nav class="nav">
+                        <ul class="nav__list">
+                            <li class="nav__item item__nav">
+                                <input
+                                    class="popup__input nav__link"
+                                    type="checkbox"
+                                    id="check"
+                                />
+                                <label for="check" class="popup__label">
+                                    <img
+                                        src="./img/pngwing.png"
+                                        width="35"
+                                        height="35"
+                                        class="nav__img-link"
+                                        alt="show menu-popup links"
```

Comment:
For decorative images implemented with `img` tag, `alt` attribute value should be an empty value.